dump978-fa/Dockerfile.template

```diff
@@ -16,34 +16,36 @@ RUN apt update && \
 
 FROM base AS buildstep
 
+# renovate: datasource=github-tags depName=osmocom/rtl-sdr versioning=loose
+ARG RTLSDR_VERSION=v2.0.2
+# renovate: datasource=github-tags depName=pothosware/SoapyRTLSDR versioning=loose
+ARG SOAPYRTLSDR_VERSION=soapy-rtl-sdr-0.3.3
```

Comment:
You may need to do a custom capture group in the renovate.json so it ignores the "soapy-rtl-sdr" prefix when looking for an updated version. 

I'll test it on my forked version of the repo in a bit. Probably not a blocker to merging though IMO.

Reply:
I conferred a bit with @Teko012 on this earlier. It is better to use commit ID in this case, as there have not been any soapy releases during the past 3-4 years, even though there are regular commits. I have not had the time to dive into Renovate yet, but hope to do so in the near future.